tests/clv/models/test_beta_geo.py

```diff
@@ -132,25 +210,33 @@ def test_numerically_stable_logp(self, frequency, recency, logp_value):
             (500, "mcmc", 0.3),
             (2000, "mcmc", 0.1),
             (10000, "mcmc", 0.055),
-            (2000, "map", 0.1),
+            (2000, "map", 0.11),
```

Comment:
Same here, let's try to pass a custom seed, so the seeding is not test-order dependent, and see if the old map expected result can stay the same

Reply:
here it's already creating new seed with each test iteration, I'm not sure how should I make it even more per test

Reply:
`self.rng` will be order dependent. You can create a seed inside the test (it's fine if the different parametrizations use the same seed)

Reply:
but it's already done like that:
    @pytest.mark.slow
    @pytest.mark.parametrize(
        "N, fit_method, rtol, seed",
        [
            (
                500,
                "mcmc",
                0.3,
            ),
            (2000, "mcmc", 0.1),
            (10000, "mcmc", 0.055),
            (2000, "map", 0.1),
        ],
    )
    def test_model_convergence(self, N, fit_method, rtol, seed, model_config):
        rng = np.random.default_rng(146)
        recency, frequency, _, T = self.generate_data(
            self.a_true, self.b_true, self.alpha_true, self.r_true, N, rng=rng
        )
        data = pd.DataFrame(
            {
                "customer_id": list(range(len(frequency))),
                "frequency": frequency,
                "recency": recency,
                "T": T,
            }
        )
        # b parameter has the largest mismatch of the four parameters
        model = BetaGeoModel(
            data=data,
            model_config=model_config,
        )
        model.build_model()

        sample_kwargs = dict(random_seed=rng, chains=2) if fit_method == "mcmc" else {}
        model.fit(fit_method=fit_method, progressbar=False, **sample_kwargs)

        fit = model.fit_result
        np.testing.assert_allclose(
            [fit["a"].mean(), fit["b"].mean(), fit["alpha"].mean(), fit["r"].mean()],
            [self.a_true, self.b_true, self.alpha_true, self.r_true],
            rtol=rtol,
        )